OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -59,6 +59,9 @@ public Player Owner
 
 			set
 			{
+				if (value == null || !(value is Player))
```

Comment:
Do you have a repro case of this being called with `!(value is Player)`?  That shouldn't be possible.